src/structures/Message.js

```diff
@@ -150,7 +150,7 @@ class Message extends Base {
      * @type {ReactionManager}
      */
     this.reactions = new ReactionManager(this);
-    if (data.reactions && data.reactions.length > 0) {
+    if (data.reactions?.length > 0) {
```

Comment:
Do we need to check the length? for-of loops short-circuit in this case.
```suggestion
    if (data.reactions) {
```